src/components/answered-question/Answer.vue

```diff
@@ -0,0 +1,64 @@
+<template>
+  <div class="grid grid-flow-col pt-2 border-b border-gray-100">
+    <div class="row-span-3 m-3 text-4xl text-gray-300">
+      <v-icon x-large>mdi-arrow-up-drop-circle</v-icon>
+
+      <p class="mt-2 mr-6 text-center">{{ answer.score }}</p>
+
+      <v-icon x-large class="pb-6">mdi-arrow-down-drop-circle</v-icon>
+    </div>
+
+    <div class="col-span-2 row-span-2 text-lg">
+      <!-- eslint-disable-next-line vue/no-v-html -->
+      <p class="pt-2 text-base" v-html="answer.body" />
+      <div class="flex justify-between col-span-1 row-span-2">
+        <div>
+          <p
+            class="inline-block p-1 mr-2 text-sm leading-none text-gray-300 bg-gray-100 rounded"
+          >share</p>
+
+          <p
```

Comment:
I'm not sure about this formatting. I think better would be:
```
<p class="inline-block p-1 mr-2 text-sm leading-none text-gray-300 bg-gray-100 rounded">
  improve this answer
</p>
```

Adjust eslint formatter, if required.

Reply:
Fixed